vcd/resource_vcd_catalog.go

```diff
@@ -33,7 +38,7 @@ func resourceVcdCatalog() *schema.Resource {
 			"name": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
+				//ForceNew: true,
```

Comment:
Should we elaborate the comment if we leave it? Like why it's commented.

Reply:
changed the comment